### DISCUSSION TITLE
store: allow explicitly opting out of wrapping

### DISCUSSION THREAD

**Discussion by otonashixav** [solidjs/solid#1022] [Ideas]
This would improve performance in cases where objects and arrays are only ever completely replaced, especially when they have a large number of keys. Currently one way to do this is to wrap the object/arrays in a class, which the store will not attempt to wrap. See https://discord.com/channels/722131463138705510/980543705440206869. 

Example implementation:
```ts
const $NOTWRAPPABLE = Symbol('solid-store-not-wrappable')
function setNotWrappable(v) {
  /*v is an object*/ && Object.defineProperty(v, $NOTWRAPPABLE, { value: true })
}
function isWrappable(v) {
  return /*v is an object*/ && !v[$NOTWRAPPABLE] && /*v is wrappable*/;
}|
```

**Comment by ryansolid** [comment]
Interesting. The check on read `isWrappable` seems trivial. Manually adding this every time seems cumbersome. But less so than making it a class.
